src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -670,11 +683,152 @@ public class DeleteOldRevisionsResult
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
 
+
+            NotifyAboutConflictRevisionsIfNeeded(conflicted, nonPersistentFlags, deleted, lowerIdPrefix);
+
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
             return result;
         }
 
+        private ConcurrentUniqueQueue<string, string> _exceededConflictsQueue = new ConcurrentUniqueQueue<string, string>(maxSize: 64);
+
+        private void NotifyAboutConflictRevisionsIfNeeded(bool conflicted, NonPersistentDocumentFlags nonPersistentFlags, long deleted, Slice lowerIdPrefix)
+        {
```

Comment:
Please do it as in NotificationCenter.Paging where there is a backgreound task that does the merge of all recent notifications and creates a notification if necessary

Reply:
Done, also added a page for the studio to show the exceeded docs

Reply:
![RavenDB-21101 Studio](https://github.com/ravendb/ravendb/assets/48368355/4d3dc378-374f-4b34-96f1-3d18bd170e29)

Reply:
Tested manually